addon/classes/Column.js

```diff
@@ -333,7 +342,7 @@ export default class Column extends Ember.Object.extend({
 
     let { subColumns } = options;
 
-    subColumns = emberArray(makeArray(subColumns).map((sc) => new Column(sc)));
+    subColumns = emberArray(makeArray(subColumns).map((sc) => merge(sc, { parent: this })).map((sc) => new Column(sc)));
     subColumns.setEach('_group', this);
```

Comment:
LOL its all good. I think `parent` works okay. Make sure you change the rest of `_group` to `parent`. I think it's referenced in a couple places.